guidelines/sc/20/parsing.html

```diff
@@ -9,6 +9,8 @@ <h4>Parsing</h4>
       duplicate attributes, and any IDs are unique, except where the specifications allow
       these features.
    </p>
+
+   <p class="note">This SC should be considered as automatically met for any content using HTML.  Modern browsers all have automatic error correction for parsing errors, and issues such as incorrect states or names due to a duplicate ID, or missing roles due to inappropriately nested elements are covered by different success criteria. This criterion can therefore be ignored as being redundant.  It no longer provides any benefit to people with disabilities in itself and should not be enforced or required for accessibility.</p>
```

Comment:
This is entirely editorial, just to match established style, as I understand it:
```suggestion
   <p class="note">This Success Criterion should be considered as automatically met for any content using HTML.  Modern browsers all have automatic error correction for parsing errors, and issues such as incorrect states or names due to a duplicate ID, or missing roles due to inappropriately nested elements are covered by different Success Criteria. This criterion can therefore be ignored as being redundant.  It no longer provides any benefit to people with disabilities in itself and should not be enforced or required for accessibility.</p>
```